federation-2/router-bridge/js-src/runtime.js

```diff
@@ -1,17 +1,13 @@
-// First we initialize the ops cache.
-// This maps op names to their id's.
-Deno.core.ops();
-
 // Then we define a print function that uses
 // our op_print op to display the stringified argument.
 const _newline = new Uint8Array([10]);
 
 function print(value) {
-  Deno.core.dispatchByName("op_print", 0, value.toString(), _newline);
+  Deno.core.print(value.toString(), _newline);
```

Comment:
now that this is in place, we can definitely expose log functions that would leverage tracing eg log.warn() log.error() log.debug() etc 🎉